runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -358,6 +358,54 @@ public void subscribeListener(@Nonnull StreamListener streamListener, @Nonnull S
                         (timestamp == null) ? getTimestamp().getSequence() : timestamp.getSequence(), bufferSize);
     }
 
+    /**
+     * Subscribe to transaction updates on all tables with the specified streamTag and namespace.
+     * Objects returned will honor transactional boundaries.
+     *
+     * @param streamListener   client listener for callback
+     * @param namespace        the CorfuStore namespace to subscribe to
+     * @param streamTag        only updates of tables with the stream tag will be polled
+     * @param timestamp        if specified, all stream updates after this timestamp will be returned,
+     *                         if null, only future updates will be returned
+     * @param bufferSize       maximum size of buffered transaction entries
+     */
+    public void subscribeListener(@Nonnull StreamListener streamListener, @Nonnull String namespace,
```

Comment:
Is there an ask to make this bufferSize configurable from any consumer of this api?
If not perhaps we can hide the bufferSize into a runtime parameter?

Reply:
Not sure, this was done by Wenbin. I see one usage in nsx (we can check if this usage is really required).